tests/HttpKeyValue/TestHttpKeyValue.py

```diff
@@ -20,7 +20,7 @@ def hurmadb_start(port):
     Start HurmaDB on given port, return an object that corresponds to the created process.
     """
     if START:
-        return subprocess.Popen(['../../hurmadb', str(port)])
+        return subprocess.Popen(['../../hurmadb -h ', str(port)])
```

Comment:
This code doesn't work. -h should be a separated list item.

Also DBMS should start on random ports for both HTTP and PgSQL (even in tests for a single protocol) since if server terminates with an error port could be not available for bind()'ing for some time.